launcher/ui/MainWindow.ui

```diff
@@ -662,7 +662,7 @@
      <normaloff>.</normaloff>.</iconset>
    </property>
    <property name="text">
-    <string>Accounts</string>
+    <string notr="true">Accounts</string>
```

Comment:
Why turn off translation for this String?

Reply:
its translated manually since it can be the name of the default account

Reply:
https://github.com/leo78913/PrismLauncher/blob/accounts-menus-fixes/launcher/ui/MainWindow.cpp#L420

Reply:
but is there a reason to turn of the translation for the string before the call to `retranslateUI` ?

Reply:
i turned it off because otherwise when changing language / translations are loaded it would update the accounts button name to "Accounts" regardless if a default account is selected

Reply:
alternatively i can just move the code that sets the button name to the selected account bellow the call to `ui->retranslateUI(this);`

Reply:
done